salt/top.sls

```diff
@@ -48,6 +48,7 @@ base:
 
   'redash':
     - redash
+    - postgres
```

Comment:
I figure this should be first?

Reply:
Doesn't matter for the redash application but swapped order so it is alphabetical.